packages/core/src/bundle.ts

```diff
@@ -267,6 +269,17 @@ function makeBundleVisitor(
     };
   }
 
+  function replaceInlineRef(node: OasRef, resolved: ResolveResult<any>, ctx: UserContext) {
```

Comment:
Rename it to `resolveBundledComponent` plz

Reply:
done